romanesco/format/graph/adjacencylist_to_networkx.py

```diff
@@ -0,0 +1,5 @@
+from networkx.readwrite.adjlist import read_adjlist
+from StringIO import StringIO
```

Comment:
Makes sense, I noticed occurrences in other places so should I go ahead and do a project-wide replace for it?